### PR TITLE
#22341 Don't add statistics to export

### LINKED ISSUE #22341: Statistics result is exported when "Show statistics tab" is Always
### Description

Run `select 1, 2, 3, 4, 5` and export the result to markdown:

|1|2|3|4|5|
|-|-|-|-|-|
|1|2|3|4|5|
|-1|select 1,2,3,4,5|Fri Jan 05 15:54:07 CST 2024|Fri Jan 05 15:54:07 CST 2024|

The result in statistic tab is appended to last row.

### DBeaver Version

Community Edition 23.3.1.202312241705

### Operating System

Windows 10

### Database and driver

Database: MySQL 5.7.34-log

Driver: MySQL Connector/J mysql-connector-j-8.2.0 (Revision: 06a1f724497fd81c6a659131fda822c9e5085b6c)

### Steps to reproduce

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by E1izabeth**
Statistics should not be added to the exported data anymore.

### CODE REVIEW

**Review by LonwoLonwo — APPROVED** [review]

File: plugins/org.jkiss.dbeaver.data.transfer/src/org/jkiss/dbeaver/tools/transfer/stream/StreamTransferConsumer.java [hunk 1]
```diff
@@ -238,7 +239,9 @@ public void fetchRow(DBCSession session, DBCResultSet resultSet) throws DBCExcep
                 targetRow[i] = value;
             }
             // Export row
-            processor.exportRow(session, resultSet, targetRow);
+            if (!(resultSet instanceof StatResultSet)) {
```
Comment (serge-rider) [inline plugins/org.jkiss.dbeaver.data.transfer/src/org/jkiss/dbeaver/tools/transfer/stream/StreamTransferConsumer.java]: Just omit rows export? I think we shoul;dnb't perform export for entire resultset.

**Review by serge-rider — CHANGES_REQUESTED** [review]

File: plugins/org.jkiss.dbeaver.data.transfer/src/org/jkiss/dbeaver/tools/transfer/stream/StreamTransferConsumer.java [hunk 2]
```diff
@@ -28,6 +28,7 @@
 import org.jkiss.dbeaver.model.exec.DBCResultSet;
 import org.jkiss.dbeaver.model.exec.DBCSession;
 import org.jkiss.dbeaver.model.fs.DBFUtils;
+import org.jkiss.dbeaver.model.impl.local.StatResultSet;
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.data.transfer/src/org/jkiss/dbeaver/tools/transfer/stream/StreamTransferConsumer.java]: Do we need this import here?

Comment (E1izabeth) [reply plugins/org.jkiss.dbeaver.data.transfer/src/org/jkiss/dbeaver/tools/transfer/stream/StreamTransferConsumer.java]: No, thanks

**Review by ShadelessFox — APPROVED** [review]

**Comment by MashaKorax** [comment]
verified

**Review by serge-rider — APPROVED** [review]

**Pull request merged**